manage-data/data-store/data-streams/failure-store.md

```diff
@@ -0,0 +1,2002 @@
+---
+applies_to:
+  stack: ga 8.19.0
+  serverless: ga 9.1.0
+---
+
+# Failure store [failure-store]
+
+Failure stores are a secondary set of indices inside a data stream dedicated to storing failed documents. Failed documents are any documents that cause ingest pipeline exceptions or have a structure that conflicts with a data stream's mappings. These failures normally cause the indexing operation to fail, returning the error message in the response. 
+
+When a data stream's failure store is enabled, these failures are instead captured in a separate index and persisted to be analysed later. Clients receive a successful response with a flag indicating the failure was redirected. Failure stores do not capture failures caused by backpressure or document version conflicts. These failures are always returned as-is since they warrant specific action by the client.
+
+## Set up a data stream failure store [set-up-failure-store]
+
+Each data stream has its own failure store that can be enabled to accept failures. By default, this failure store is disabled and any ingestion problems are raised in the response to write operations.
+
+### Set up for new data streams [set-up-failure-store-new]
+
+You can specify on a data stream's template if it should enable the failure store when it is first created. The `data_stream_options` field in a [template](../templates.md) contains the settings required to enable a data stream's failure store.
```

Comment:
```suggestion
You can specify in a data stream's [template](../templates.md) if it should enable the failure store when it is first created.
```
I'd put the link on the first instance of "template" and then we can remove the second sentence, since I think it's pretty similar to what you have above the example.